fancy-team-server/server.ts

```diff
@@ -1,6 +1,6 @@
 'use strict';
-import express from 'express';
-import cors from 'cors';
+import * as express from 'express';
+import * as cors from 'cors';
```

Comment:
please sync against master  - npm i and then try tsc to see if we could avoid this change

Reply:
Works with what's in master, so I have removed my change